src/core/reactor.cc

```diff
@@ -3935,6 +3936,7 @@ void smp::configure(const smp_options& smp_opts, const reactor_options& reactor_
         nr_cpus = cpu_set.size();
     }
     smp::count = nr_cpus;
+    logger::set_shard_field_width(std::trunc(std::log10(smp::count) + 1));
```

Comment:
Off-by-one here. If smp::count == 10, the shards are numbered {0, 1, ..., 9} and field width of 2 is excessive